### ISSUE TITLE
Advantages of vector search over normal keyword+symantic search

### ISSUE THREAD

**Issue by amarmohax** [Azure/cognitive-search-vector-pr#45]
Hi,

As part of this project, did anyone compare how the vector search performs in comparison to normal keyword search+symantic option in Azure Cognitive Search?
Especially in following areas.
1. Relevance of search results - better or worse
2. Performance or elapsed time - better or worse. 
3. Use cases where keyword+symantic is better and where Vector is better.

Curious to know in case someone has already done it.
--regards.

**Comment by lppier** [comment]
I am also super curious about this too!

**Comment by farzad528** [comment]
We are actively working on our experiments and performance benchmarks. Stay tuned!

**Comment by smharvey** [comment]
Yes. Excited to know. We just got started and are also very curious. We can
report some findings too.

**Comment by wXw2007** [comment]
I have a demo session for our team, has benchmark, and many search experiments: https://microsoft-my.sharepoint.com/:v:/p/xiaowwan/ETO6SCLEV6NPgyFZfzGhkScBiSUQxs06I_ZbPl0wR5Z_3w?e=8yylcT

**Comment by amarmohax** [comment]
Can't access the above sharepoint link, is there anyway we can see it?
